test/lib/RegistryEntryCollection.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var path = require('path');
+var glob = require('glob');
+var mout = require('mout');
+var forOwn = mout.object.forOwn;
+var FILE_EXTENSION = /[^.]*$/i;
+var PACKAGE_EXPECTATION = /(.*)@(.*).json$/;
+
+/**
+ * @constructor RegistryEntry
```

Comment:
OverridesList seems more appropriate because the module does not represent a file itself is just a list of strings